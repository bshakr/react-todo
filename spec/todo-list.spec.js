import React, { Component } from 'react';
import renderer from 'react-test-renderer';
import TodoList from '../src/todo-list';
import TodoItem from '../src/todo-item';

describe('empty', () => {
  it('works', () => {
    const todos = [{text: 'Buy milk', isComplete: false }];
    function complete (){};
    const todoList = renderer.create(<TodoList todos={todos} complete={complete} />);
    let tree = todoList.toJSON();
    expect(tree).toMatchSnapshot();
  });
});

