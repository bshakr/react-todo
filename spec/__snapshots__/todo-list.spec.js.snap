exports[`empty works 1`] = `
<div
  className="row">
  <div
    className="col-sm-10 col-sm-offset-1">
    <ul
      className="todo-list">
      <li
        className="todo-item">
        <input
          checked={false}
          onChange={[Function]}
          type="checkbox" />
           
        Buy milk
      </li>
    </ul>
  </div>
</div>
`;
